core-web/libs/sdk/experiments/tsconfig.json

```diff
@@ -5,7 +5,8 @@
         "allowJs": false,
         "esModuleInterop": false,
         "allowSyntheticDefaultImports": true,
-        "strict": true
+        "strict": true,
+        "moduleResolution": "bundler"
```

Comment:
When we run the command `npx nx build sdk-experiments`, the project also builds `@dotcms/client`, `@dotcms/react`, and `@dotcms/uve` due to the declaration in the `peerDependencies`.  

To ensure that `TypeScript` supports `"imports"` and `"exports"` from `package.json` at build time, we need to set `"moduleResolution": "bundler"`.

More information in [Module Resolution - moduleResolution](https://www.typescriptlang.org/tsconfig/#moduleResolution)